currency_monthly_rate/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_res_currency_rate_monthly,access_res_currency_rate_monthly,model_res_currency_rate_monthly,monthly_rates_group,1,1,1,1
```

Comment:
What about copy the permissions from base module for the inherited model?

"access_res_currency_rate_group_all","res_currency_rate group_all","model_res_currency_rate",,1,0,0,0
"access_res_currency_rate_group_system","res_currency_rate","group_system","model_res_currency_rate","group_system",1,1,1,1

Reply:
Just added a read permission to everyone, thanks !